### ISSUE TITLE
Small "nitpicking" to make program simpler and more consistent

### ISSUE THREAD

**Issue by igor2x** [pwsafe/pwsafe#1104]
Password Safe v1.18.2 on Ubuntu 23.10. I know to some of people this small issues may look trivial, but getting through them all over and over again it would be nice if this could be fixed. You know beauty is feature.  :-)

**1. Branding**
On Password Safe on Windows the icon is always red, that is official icon of program. In Linux this official icon is black. I see no need for that and the main branding of product is lost. I suggest to replace black icon with red icon.
![image](https://github.com/pwsafe/pwsafe/assets/10895030/9c1b640c-2d65-4ae3-8731-1cffaa479ac8)

**2. Main window several small issues**
a) At top left corner there is small black icon. There should be no icon at all. Looking at other Linux programs and there is no icon in any of the program. Icon somehow indicates that you can click on it and some action will follow, but nothing happens clicking on icon. I suggest to remove the icon, because it is redundant.
b) In the middle of title bar: In the programs I have look at (e.g. text editors, image editors,...) the order of current opened file and the program itself is first opened document and then program name so: "pwsafe.psafe3 - Password Safe". And another one is this name concatenation "PasswordSafe" instead of "Password Safe". It looks like a bug, someone forgot to add a space, also the name of the program it should be clear what it is. The branding of program should be clear what it is and used consistently, to reduce confusion. Smaller confusion, simpler to use a program.
c) bottom right the URL is something very unusual and distracting. I suggest to remove it. Less distractions simpler to use a program. This kind of info belongs to Help menu.

![image](https://github.com/pwsafe/pwsafe/assets/10895030/f7ba8965-2f69-4659-9c16-5138e9e231df)

**3. Remove non-essential elements**
Remove anything that is distraction, to simplify the usage and reduce confusion. I see no need for icon there inside in the middle of the program. For other wording to make program simpler I have already written suggestions in [Yes/No buttons issue.](https://github.com/pwsafe/pwsafe/issues/1020)

![image](https://github.com/pwsafe/pwsafe/assets/10895030/395ef8a2-0286-46ba-8b29-42a0852259b4)

**4. More love to the About page**
Like it is now Help | About is little bit crowded. Left side of image is current image, right site is my suggestion.
a) At the top there is the most important part. The name of the program is just too small and the "branding" name is lacking space between Password and Safe.
b) All of the lines it would look better if they are aligned to the middle vs. current left.
c) The old "here" link problem. Links should never have "here", but instead have real text. If someone only what he or she reads is text in blue it should be immediately clear what will happen. "Here" doesn't do that. It is better to add "Check" word or similar.
d) Also "Please" word looks like begging people. Add text like "For more info..." that adds why should people visit web site.
e) There is text inviting people to read license, but it would help people much more if link to the license is provided.
f) At left side it is black icon. Password Safe branding is red icon.
g) There are three separate groups of information, but they are all blended together. On right site I have added some spacing between those three groups to make is visually easier to distinguish the logic between three groups.
h) Close button would be nice to be aligned, like middle of the window.
![image](https://github.com/pwsafe/pwsafe/assets/10895030/3397048f-9496-4e78-be6b-95961d964f0d)

In my humble opinion, I have not seen a programmer that pays attention to looks and feels. This is actually reasonable because programmer spends 99% of time on background code and 1% to provide visual feedback to user. But user spends 99% in GUI visual interface and 1% understands the logic behind. It is in my humble opinion vital to have simple to use and nice looking program. Two main features:
- simplicity is a feature (remove clutter, long too complicated sentences,...) 
- beauty is a feature (group things together, smaller/bigger fonts, branding, aliment,...)

I think Password Safe is excellent peace of program, but what maybe needs it is little more touch of a designer.

@ronys are you interested in this kind of simplifying and beautifying the program. If yes, I am more then willing to help, but if not, I don't want to waste your time.

**Comment by ronys** [comment]
Yes, I'm very much in favor of cleaning things up. Any help gratefully accepted, doubly so for pull requests.
Re the specific points you raise:

- Strictly speaking, the icon color is dark red, specifically RGB 128,0,0
- The URL in the bottom is a "placeholder" until there's something more useful to display, e.g., "password copied to clipboard". I think it looks better that a blank space.
- Help about: We've just split "(wxWidgets)" into a separate line with the wxWidgets version number, e.g., "Built with wxWidgets 2.3.4". I don't think center alignment is better, to me it makes the information less accessible.
- Other than that, I agree with your suggestions - Thanks!

**Comment by igor2x** [comment]
On left my original suggestion and on right I changed according to your suggestion. I aslo changed icon color to RGB 128,0,0. Yes left aliment looks little bit better.

![image](https://github.com/pwsafe/pwsafe/assets/10895030/472d95cc-cfdc-4bcc-9611-71836e7e2c0c)

I am not much of a programmer to provide PRs. I don't know how difficult is to establish such environment and work with it. If it is not too hard, I am willing to try out, but it has to be on Linux, because I don't have Windows box at home. How should we communicate better (but tracker is not the best way to do this).

**Comment by ronys** [comment]
Drop me a note via https://pwsafe.org/contact.php to establish a more efficient communication channel.

**Comment by igor2x** [comment]
**Status bar - part 2**
On left current design and white "canvas" color when R/W on right my suggestion (remove status bar) and when R/W then white canvas like on left site and when R-O then make canvas color the same color as toolbar. From canvas color it is immediately clear that R-O is turned on (for users that use this functionality like me in 99% of usage).

![image](https://github.com/pwsafe/pwsafe/assets/10895030/3356e115-de3b-462c-b765-ba5daa0108cf)

**Comment by igor2x** [comment]
Fixes: First post in this issue "1. Branding" is fixed in: https://github.com/pwsafe/pwsafe/commit/9aaf15a1513c1b37609e17ee2f3a60d7023941e1
